app/core/src/main/java/com/fsck/k9/FontSizes.kt

```diff
@@ -0,0 +1,109 @@
+package com.fsck.k9
+
+import android.util.TypedValue
+import android.widget.TextView
+import com.fsck.k9.preferences.Storage
+import com.fsck.k9.preferences.StorageEditor
+
+/**
+ * Manage font size of the information displayed in the message list and in the message view.
+ */
+class FontSizes {
+    var messageListSubject: Int
+    var messageListSender: Int
+    var messageListDate: Int
+    var messageListPreview: Int
+    var messageViewAccountName: Int
+    var messageViewSender: Int
+    var messageViewRecipients: Int
+    var messageViewSubject: Int
+    var messageViewDate: Int
+    var messageViewContentAsPercent: Int
+    var messageComposeInput: Int
+
+    init {
```

Comment:
Out of curiosity: Why did you separate the declaration and initialization of these properties?